casepro/backend/tests/test_junebug.py

```diff
@@ -13,20 +13,197 @@ def setUp(self):
+                            "msisdn": {
+                                "+1234": {}
+                            },
+                        },
+                        "preferred_langage": "eng_NG",
+                    },
+                    "communicate_through": None,
+                    "operator": None,
+                    "created_at": "2016-03-14T10:21:00.258406Z",
+                    "created_by": 1,
+                    "updated_at": None,
+                    "updated_by": None
+                }
+            ]
+        }
+        return (201, headers, json.dumps(resp))
+
+    def identity_store_updated_identity_callback(self, request):
+        headers = {'Content-Type': 'application/json'}
+        resp = {
+            "count": 1,
+            "next": None,
+            "previous": None,
+            "results": [
+                {
+                    "id": "test_id",
+                    "version": 1,
+                    "details": {
+                        "name": "test",
+                        "addresses": {
+                            "msisdn": {
+                                "+1234": {}
+                            },
+                        },
+                        "preferred_langage": "eng_NG",
+                    },
+                    "communicate_through": None,
+                    "operator": None,
+                    "created_at": "2016-02-14T10:21:00.258406Z",
+                    "created_by": 1,
+                    "updated_at": "2016-03-14T10:21:00.258406Z",
+                    "updated_by": 1
+                }
+            ]
+        }
+        return (201, headers, json.dumps(resp))
+
+    @responses.activate
+    def test_pull_contacts_recently_created(self):
         Contact.objects.all().delete()
 
+        self.add_identity_store_callback(
+            'created_at__lte=2016-03-14T10%3A21%3A00&created_at__gte=2016-03-14T10%3A25%3A00',
+            self.identity_store_created_identity_callback
+        )
+
+        self.add_identity_store_callback(
+            'updated_at__lte=2016-03-14T10%3A21%3A00&updated_at__gte=2016-03-14T10%3A25%3A00',
+            self.identity_store_no_matches_callback
+        )
+
+        self.add_identity_store_callback(
+            'updated_at__lte=2016-03-14T10%3A21%3A00&updated_at__gte=2016-03-14T10%3A25%3A00&'
+            'optout__optout_type=forget',
+            self.identity_store_no_matches_callback
+        )
+
         (created, updated, deleted, ignored) = self.backend.pull_contacts(
-            self.unicef, None, None)
+            self.unicef, '2016-03-14T10:25:00', '2016-03-14T10:21:00')
+        self.assertEqual(created, 1)
+        self.assertEqual(updated, 0)
+        self.assertEqual(deleted, 0)
+        self.assertEqual(ignored, 0)
+        self.assertEqual(Contact.objects.count(), 1)
+
+    @responses.activate
+    def test_pull_contacts_recently_updated(self):
+        Contact.objects.all().delete()
```

Comment:
Are `Contact` objects not cleaned up in between each test automatically? Or are there some fixtures that are automatically loaded that we don't want?